lib/core/wireprotocol/command.js

```diff
@@ -90,6 +91,18 @@ function _command(server, ns, cmd, options, callback) {
   const inTransaction = session && (session.inTransaction() || isTransactionCommand(finalCmd));
   const commandResponseHandler = inTransaction
     ? function(err) {
+        // We need to add a TransientTransactionError errorLabel, as stated in the transaction spec.
+        if (
+          err &&
+          err instanceof MongoNetworkError &&
+          !err.hasErrorLabel('TransientTransactionError')
+        ) {
+          if (err.errorLabels == null) {
```

Comment:
nit: let's initialize `errorLabels` inside of `MongoError`, so we can be sure that it is always there.

Reply:
It looks like initializing `errorLabels` inside of `MongoError` causes some transactions tests to fail. I spent a little while investigating and couldn't figure it out, but I didn't want to make this work depend on that issue.